frontend/src/components/navbar/navstyle.module.scss

```
@@ -1,40 +1,59 @@
-.toolBar{
-    position:fixed;
-    width: 100%;
-    height: 56px;
-    background: transparent;
-    top:0;
-    left:0;
-    z-index: 1;
-}
+@import url("https://fonts.googleapis.com/css?family=Oswald&display=swap");
 
-.toolBarNav{
-    display:flex;
-    align-items: center;
-    height: 100%;
-    padding:0 1rem;
+.toolBar {
+  position: absolute;
+  width: 100%;
+  height: 70px;
+  background: transparent;
+  top: 0;
+  left: 0;
+  z-index: 1;
 }
 
-.toolBarItems ul{
-    list-style: none;
-    display:flex;
+.toolBarNav {
+  display: flex;
+  align-items: center;
+  height: 100%;
+  padding: 0 1rem;
 }
 
-.toolBarRef{
-    text-decoration: none;
-    padding:20px 10px 20px 10px;
-    color: rgb(255, 166, 0);
+.toolBarItems ul {
+  list-style: none;
+  display: flex;
 }
 
-.toolBarRefActive{
-    border-bottom: 1px solid rgb(255, 166, 0);
-    border-image-slice: 1;
+.toolBarRef {
+  text-decoration: none;
+  padding: 20px 10px 20px 10px;
+  color: rgb(255, 255, 255);
+  position: relative;
+  font-family: "Oswald", sans-serif;
+}
+.toolBarRef:hover {
+  color: rgb(217, 176, 129);
+  text-decoration: none;
+}
+.toolBarRef::before {
+  content: "";
+  position: absolute;
+  width: 100%;
+  height: 2px;
+  bottom: -3px;
+  left: 0;
+  background-color: rgb(217, 176, 129);
+  visibility: hidden;
+  transform: scaleX(0);
+  transition: all 0.3s ease-in-out;
+}
+.toolBarRef:hover:before {
```

Mantén los espacios entre las declaraciones: 
```suggestion
}

.toolBarRef:hover {
  color: rgb(217, 176, 129);
  text-decoration: none;
}

.toolBarRef::before {
  content: "";
  position: absolute;
  width: 100%;
  height: 2px;
  bottom: -3px;
  left: 0;
  background-color: rgb(217, 176, 129);
  visibility: hidden;
  transform: scaleX(0);
  transition: all 0.3s ease-in-out;
}

.toolBarRef:hover:before {
```